### PR TITLE
fixed and optimised docker deployment

### PR DESCRIPTION
**Pull request by rugged-bl**
+ Linux is case-sensitive @NikitaVirachev :)
+ Every RUN creates extra unrequired layers
+ Docker couldn't find babel config

### CODE REVIEW

**Comment by ASlepushko** [comment]
You are breathtaking! 🐱‍🏍

**Review by prometium — APPROVED** [review]
It was not necessary to change dockerignore, since all files and directories are copied selectively.
Other fixes are very nice.

File: .dockerignore [hunk 1]
```diff
@@ -3,3 +3,7 @@ node_modules
 
 # production
 dist
+
+.git
+.idea
+__mocks__
```
Comment (ASlepushko) [inline .dockerignore:9]: I'm not sure that `__mock__` should be ignored

**Review by ASlepushko — COMMENTED** [review]
Since we don't run tests in the Docker so okay

**Pull request merged**